### DISCUSSION TITLE
Idea: Lists inside Lists, or Categories -> Lists

### DISCUSSION THREAD

**Discussion by NeussConsulting** [community/community#18567] [Other Feature Feedback, Questions, & Ideas]
I love the new Lists Feature, but I would love to see on more level of depth.

For example, I'm Software Developer and I love many projects out there to use in my projects.

So I want to have a list/categorie "3rd Party Libs".
But there are 100 of libs in totaly different categories:
- Databases
- Security
- Rest
- Math
- ...

To organize better, one more level would be cool.
Sure you can do it with creating a list for every category like "3rd Party - Databases", "3rd Party - Security" but that would not be my style.

**Comment by Agoines** [comment]
I think it's cool

**Comment by RobyRew** [comment]
Yes, this is great for subcategories 
For example having. Windows and inside having security, tweaking, inside of tweaking having GUI apps CLI apps or for example not maintained and maintained Repos

**Reply by Chanvannak746** [reply]
[]()@ajs256

**Reply by Chanvannak746** [reply]
> []()@ajs256 

វត្តកំបូលបានខ្ពស់

**Comment by Danyalkasiri** [comment]
> I love the new Lists Feature, but I would love to see on more level of depth.
> 
> For example, I'm Software Developer and I love many projects out there to use in my projects.
> 
> So I want to have a list/categorie "3rd Party Libs".
> But there are 100 of libs in totaly different categories:
> - Databases
> - Security
> - Rest
> - Math
> - ...
> 
> To organize better, one more level would be cool.
> Sure you can do it with creating a list for every category like "3rd Party - Databases", "3rd Party - Security" but that would not be my style.

**Comment by kimkumsook-g** [comment]
> I love the new Lists Feature, but I would love to see on more level of depth.
> 
> For example, I'm Software Developer and I love many projects out there to use in my projects.
> 
> So I want to have a list/categorie "3rd Party Libs".
> But there are 100 of libs in totaly different categories:
> - Databases
> - Security
> - Rest
> - Math
> - ...
> 
> To organize better, one more level would be cool.
> Sure you can do it with creating a list for every category like "3rd Party - Databases", "3rd Party - Security" but that would not be my style.

**Comment by Zer0F8th** [comment]
I second this. Having nested lists to better organize starred repos would be amazing! 

A list structure that could look like the following would be great:
- Cyber Security
  - Active Directory
    - [BloodHound](https://github.com/BloodHoundAD/BloodHound)
 - ...
   - ...
     - ...

Bumping for visibility.

**Comment by FlurryNight** [comment]
+1, Was about to open a discussion too

**Comment by WilsonCWong** [comment]
This would be awesome to have.

**Comment by johngagefaulkner** [comment]
I came here to post this suggestion, glad I searched first. Bumping the maximum amount of lists from 16 → 32 was a welcome change (obviously) but "Lists" are effectively in-app bookmarks and not being able to create "sub-lists" (aka: folders and subfolders, like in your browser's bookmarks/favorites) makes this super limiting for me and, I'd imagine, the vast majority of people using Lists.
